lib/shared/store.js

```diff
@@ -28,6 +28,52 @@ const fileExtensions = require('./file-extensions')
 const utils = require('./utils')
 const packageJSON = require('../../package.json')
 
+/**
+ * @summary Verify and throw if any state fields are nil
+ * @function
+ * @public
+ *
+ * @param {Object} object - state object
+ * @param {Array<Array<String>> | Array<String>} fields - array of object field paths
+ * @param {String} name - name of the state we're dealing with
+ * @throws
+ *
+ * @example
+ * const fields = [ 'type', 'percentage' ]
+ * verifyNoNilFields(action.data, fields)
+ */
+const verifyNoNilFields = (object, fields, name) => {
```

Comment:
Is it worth creating a standalone unit-test for this function? Or are the existing unit-tests sufficient?
(I note for example that there's nothing testing that multiple fields are missing at once, which I guess means the `.join(', ')` part isn't being tested?)

Reply:
That's true, I'll add tests for multiple missing fields.